sequence/sequences/compose/arith.py

```diff
@@ -0,0 +1,74 @@
+from sequence.core.core import Sequence
+
+import numpy as np
+
+
+class Arith(Sequence):
```

Comment:
perhaps `ArithmeticOperations` or `IntegerSequenceOperations` ?

In fact, the method `is_finite` is correct just for integer sequences.

For example, take the two non-integer sequences `(-1)^n` and `(-1)^(n+1)`. Both are infinite, but their sum is finite, as it is the constant sequence 0

Reply:
> In fact, the method `is_finite` is correct just for integer sequences.

Do we have the same definition of "finite"? Would appreciate if you could link the one you are using.

I thought this property meant that the sequence is of finite length. "finite sequence" afaik is a standing term with this meaning. So, the constant sequence with a countably infinite number of terms, according to this definition, would not be finite, but infinite.

https://en.wikipedia.org/wiki/Sequence#Finite_and_infinite

Reply:
re name, I prefer short ones, otherwise object string identifiers become long and unreadable quickly... how about `ArithOp`?

Reply:
> Do we have the same definition of "finite"? Would appreciate if you could link the one you are using.
> 
> I thought this property meant that the sequence is of finite length. "finite sequence" afaik is a standing term with this meaning. So, the constant

I see your point, and I understand the misunderstanding between us. Perhaps, I expressed myself bad. Let re-explain the situation:
Suppose we have two sequences a_n and b_n, and we want to add them. We have 3 cases:
1. both are infinite
2. one is finite and the other is infinite
3. both are finite

In the first case, no problem.
In the second case, how should be defined the sum of a finite sequence with an infinite one?
in the third case we have two subclasses:
- if the two sequences have the same length -> no problem
- if the two sequences have different length -> how it is defined the addition?

Possible solutions:
- you can just add infinite sequences or sequences with the same length
- we consider the finite sequence as the sub-vector space of all sequence which have compact support in the vector space of all sequences ( [REF](https://en.wikipedia.org/wiki/Sequence_space#c,_c0_and_c00))

Both solution have pro and contro.

What do you think?

Reply:
I think the best option is to take the minimum of lengths, for the terms where the combination is defined.